### PR TITLE
Add option to disable qt hyperdiffusion

### PR DESCRIPTION
**Pull request by LenkaNovak**
# PULL REQUEST

## Purpose and Content
- adds an option to disable hyperdiffusion of qt 

## Benefits and Risks
- benefit: adds stability (similar timestep compared to the dry run)
- risk: this is not the final solution (may need to adjust `e_tot` or use limiters in the future)

## Linked Issues
(Provide references to any link issues. Use closes #issuenum to automatically close an open issue)
- Fixes #
- Closes #

## PR Checklist
- [x] This PR has a corresponding issue OR is linked to an SDI.
- [x] I have followed CliMA's codebase [contribution](https://clima.github.io/ClimateMachine.jl/latest/Contributing/) and [style](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) guidelines OR N/A.
- [x] I have followed CliMA's [documentation policy](https://github.com/CliMA/policies/wiki/Documentation-Policy).
- [x] I have checked all issues and PRs and I certify that this PR does not duplicate an open PR.
- [x] I linted my code on my local machine prior to submission OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code used in an integration test OR N/A.
- [x] All tests ran successfully on my local machine OR N/A.
- [x] All classes, modules, and function contain docstrings OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: examples/hybrid/hyperdiffusion.jl [hunk 1]
```diff
@@ -37,9 +37,12 @@ function hyperdiffusion_tendency_clima!(Yₜ, Y, p, t)
         ᶜρ = Y.c.ρ
         ᶜuₕ = Y.c.uₕ
         (; ᶜp, ᶜχ, ᶜχuₕ) = p # assume ᶜp has been updated
-        (; ghost_buffer, κ₄, divergence_damping_factor, use_tempest_mode) = p
+        (; ghost_buffer, κ₄, divergence_damping_factor, use_tempest_mode, disable_qt_hyperdiffusion) = p
         point_type = eltype(Fields.local_geometry_field(axes(Y.c)).coordinates)
         is_ρq_tot = :ρq_tot in propertynames(Y.c)
+        if is_ρq_tot && disable_qt_hyperdiffusion
+            is_ρq_tot = false # TODO: reconcile this with ρe_tot or remove if instability fixed with limiters
+        end
```
Comment (charleskawczynski) [inline examples/hybrid/hyperdiffusion.jl:45]:
```suggestion
        # TODO: reconcile this with ρe_tot or remove if instability fixed with limiters
        is_ρq_tot = :ρq_tot in propertynames(Y.c) && !disable_qt_hyperdiffusion
```

Is this maybe clearer? Better to define variables once, I think.

**Comment by szy21** [comment]
> @szy21 does this look good to you? (I will format it later...) Also, how does one trigger the long run pipeline to test this?

To set up a long run, add something like this in .buildkite/longruns/pipeline.yml
- label: ":computer: held suarez (ρe_tot) equilmoist noqthyper"
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 400secs --disable_qt_hyper true --t_end 600days --fps 30 --job_id longrun_hs_rhoe_equil_filter --dt_save_to_sol 6hours --dt_save_to_disk 10days"
        artifact_paths: "longrun_hs_rhoe_equil_noqthyper/*"
        agents:
          slurm_cpus_per_task: 8

File: .buildkite/longruns/pipeline.yml [hunk 2]
```diff
@@ -141,4 +141,10 @@ steps:
         artifact_paths: "longrun_hs_rhoe_equil_dt250/*"
         agents:
           slurm_cpus_per_task: 8
+      
+      - label: ":computer: held suarez (ρe_tot) equilmoist noqthyper"
+        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 400secs --disable_qt_hyper true --t_end 1000days --fps 30 --job_id longrun_hs_rhoe_equil_noqthyper --dt_save_to_sol 6hours --dt_save_to_disk 10days"
```
Comment (szy21) [inline .buildkite/longruns/pipeline.yml:146]:
```suggestion
        command: "julia --color=yes --project=examples --threads=8 examples/hybrid/driver.jl --forcing held_suarez --moist equil --vert_diff true --microphy 0M --dt 400secs --disable_qt_hyperdiffusion true --t_end 1000days --fps 30 --job_id longrun_hs_rhoe_equil_noqthyper --dt_save_to_sol 6hours --dt_save_to_disk 10days"
```
(sorry, a typo in the previous comment)

**Review by szy21 — APPROVED** [review]

File: .buildkite/longruns/pipeline.yml (hunk 2 above)
Comment (LenkaNovak) [reply .buildkite/longruns/pipeline.yml:146]: oops! good catch! 🐛

**Comment by szy21** [comment]
Actually, could we make `disable_qt_hyperdiffusion` a hyperdiffusion cache, since it is only used by hyperdiffusion? I can push the fix as well if you don't mind.

**Comment by LenkaNovak** [comment]
Sure, it's not going to be a permanent fix, so I wouldn't worry too much about where exactly it lives. But feel free to add it, if you find it more useful in the hyper cache. :)

**Comment by szy21** [comment]
> Sure, it's not going to be a permanent fix, so I wouldn't worry too much about where exactly it lives. But feel free to add it, if you find it more useful in the hyper cache. :)

Yeah, I just think it would be better organized and it's a very easy fix. I can also go ahead and merge it after the fix if that sounds good to you:)

**Comment by LenkaNovak** [comment]
Sounds good, thank you!

**Pull request merged**